package.json

```diff
@@ -33,7 +33,7 @@
         "deploy-self-hosted:optimism-kovan-dev1": "npm run codegen:optimism-kovan-dev1 && graph deploy -l ${npm_package_version} --node http://127.0.0.1:8020 --ipfs https://ipfs.network.thegraph.com/ipfs/ perpetual-protocol/perpetual-v2-optimism-kovan-dev1",
         "deploy-self-hosted:optimism-kovan-dev2": "npm run codegen:optimism-kovan-dev2 && graph deploy -l ${npm_package_version} --node http://127.0.0.1:8020 --ipfs https://ipfs.network.thegraph.com/ipfs/ perpetual-protocol/perpetual-v2-optimism-kovan-dev2",
         "deploy-self-hosted:optimism-goerli": "npm run codegen:optimism-goerli && graph deploy -l ${npm_package_version} --node http://127.0.0.1:8020 --ipfs https://ipfs.network.thegraph.com/ipfs/ perpetual-protocol/perpetual-v2-optimism-goerli",
-        "deploy-self-hosted:optimism": "npm run codegen-self-hosted:optimism && graph deploy -l ${npm_package_version} --node http://127.0.0.1:8020 --ipfs https://ipfs.network.thegraph.com/ipfs/ perpetual-protocol/perpetual-v2-optimism",
+        "deploy-self-hosted:optimism": "npm run codegen:optimism && graph deploy -l ${npm_package_version} --node http://127.0.0.1:8020 --ipfs https://ipfs.network.thegraph.com/ipfs/ perpetual-protocol/perpetual-v2-optimism",
```

Comment:
why does `deploy-self-hosted:optimism` command run `codegen:optimism` instead of `codegen-self-hosted:optimism`

Reply:
`codegen-self-hosted:optimism` will do the `jq` replace to let self-hosted graph node use `baseForSelfHosted`, but since I remove `baseForSelfHosted` from the config, so the command will fail.

Instead of fixing the command, I choose to ignore it for now (aka 偷懶), simply change to `npm run codegen:optimism`.